### ISSUE TITLE
Fail to execute XLA_TARGET=cuda111 on CUDA 12.0 on Linux

### ISSUE THREAD

**Issue by zacky1972** [elixir-nx/nx#1046]
Hi,

I run `XLA_TARGET=cuda111 EXLA_TARGET=cuda mix run -r bench/softmax.exs` on EXLA with CUDA 12.0 on Linux, but I got the following error:

```
$ XLA_TARGET=cuda111 EXLA_TARGET=cuda mix run -r bench/softmax.exs 

05:38:28.958 [info] Found a matching archive (xla_extension-x86_64-linux-cuda111.tar.gz), going to download it
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  213M  100  213M    0     0  13.2M      0  0:00:16  0:00:16 --:--:-- 16.5M

05:38:45.087 [info] Successfully downloaded the XLA archive
Unpacking /home/zacky/.cache/xla/0.4.0/cache/download/xla_extension-x86_64-linux-cuda111.tar.gz into /home/zacky/github/nx/exla/cache
g++ -fPIC -I/home/zacky/.asdf/installs/erlang/25.2/erts-13.1.3/include -Icache/xla_extension/include -O3 -Wall -Wno-sign-compare -Wno-unused-parameter -Wno-missing-field-initializers -Wno-comment -shared -std=c++17 -w -DLLVM_ON_UNIX=1 c_src/exla/exla.cc c_src/exla/exla_nif_util.cc c_src/exla/exla_client.cc -o cache/libexla.so -Lcache/xla_extension/lib -lxla_extension -Wl,-rpath,'$ORIGIN/lib'

05:39:26.210 [info] domain=elixir.xla file=tensorflow/compiler/xla/service/service.cc line=173  XLA service 0x7ff470021c20 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:

05:39:26.210 [info] domain=elixir.xla file=tensorflow/compiler/xla/service/service.cc line=181    StreamExecutor device (0): NVIDIA GeForce RTX 3060, Compute Capability 8.6

05:39:26.210 [info] domain=elixir.xla file=tensorflow/compiler/xla/pjrt/gpu/se_gpu_pjrt_client.cc line=194  Using BFC allocator.

05:39:26.210 [info] domain=elixir.xla file=tensorflow/compiler/xla/pjrt/gpu/gpu_helpers.cc line=105  XLA backend allocating 9885817241 bytes on device 0 for BFCAllocator.
Segmentation fault (core dumped)
```

**Comment by josevalim** [comment]
Correct CUDA 1.12 has breaking changes and afaik XLA/Tensorflow do not yet support it. You can try cuda118 too, but that likely won’t work either.

**Comment by josevalim** [comment]
The solution is to downgrade your system cuda until a new XLA version is out.

**Comment by zacky1972** [comment]
I see.

**Comment by Joss-Steward** [comment]
It would be helpful to add a note or warning about the supported versions of CUDA to the documentation of BumbleBee, Nx, or somewhere else that's easily visible and appropriate.  I've spent several hours now trying to get LiveBook with Kino-BumbleBee working on WSL and finally traced the last issue down to CUDA 1.12 being installed by default while following the instructions to set up CUDA on linux.

**Comment by polvalente** [comment]
Supported CUDA versions are listed in the `xla` [README](https://github.com/elixir-nx/xla).
I don't see any reason to keep this issue open. WDYT @josevalim?
